src/index.html

```diff
@@ -1,22 +1,62 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900&display=swap" rel="stylesheet" />
     <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
-    />
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Moyo header</title>
-    <link
-      rel="stylesheet"
-      href="./style.css"
-    />
+    <link rel="stylesheet" href="./style.css" />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <img src="images/logo.png" alt="logo" />
+      <nav>
+        <ul>
+          <li class="li_nav apple">
+            <a href="#apple" class="nav__link a1 is-active">
+                Apple
+            </a>
+          </li>
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav_item">
            <a href="#apple" class="nav__link is-active">
                Apple
            </a>
          </li>
```
add classes for all elements and style by them.
a1, a2, a3, a4... classes it is hardcode. you don't need them.

acive link should have is-acive class. it is enough to style by design.